django_components/component.py

```diff
@@ -32,8 +34,20 @@ def template(self, context):
         raise NotImplementedError("Missing template() method on component")
 
     def render_dependencies(self):
+        """Helper function to access media.render()"""
```

Comment:
Interesting addition of docstrings across this PR. I've never used them in a project before, I just tend to keep functions small so they don't need them. But let's try and keep them, might make things easier for people wanting to help!

Reply:
Cool. Yeah it is more a force of habit for me. I tend to add them everywhere even when they may not be absolutely necessary.